test/unit/package_test.rb

```diff
@@ -233,158 +233,158 @@ def test_centimetres_girth_cylinder
 
   end
 
-  def test_centimetres_girth
+  test "#centimetres_girth" do
     refute_predicate @package, :cylinder?
     assert_in_delta 22, @package.centimetres(:girth), 1
     assert_in_delta 22, @package.centimetres(:around), 1
     assert_in_delta 22, @package.centimetres(:circumference), 1
   end
 
-  def test_centimetres_volume_cylinder
+  test "#centimetres for the volume of acylinder" do
     @package = Package.new(@weight, @dimensions, @options.merge(cylinder: true, units: :metric, dim_units: :metric))
 
     assert_predicate @package, :cylinder?
     assert_in_delta 522.4, @package.centimetres(:volume), 1
   end
 
-  def test_centimetres_volume
+  test "#centimetres for volume" do
     refute_predicate @package, :cylinder?
     assert_equal 210, @package.centimetres(:volume)
   end
 
-  def test_centimetres_box_volume
+  test "#centimetres for box_volume" do
     assert_equal 210, @package.centimetres(:box_volume)
   end
 
-  def test_centimetres_unknown
+  test "#centimetres of an unknown value" do
     assert_nil @package.centimetres(:unknown)
   end
 
-  def test_centimetres_alias_cm
+  test "#centimetres alias to cm" do
     assert_equal @dimensions, @package.centimetres
     assert_equal @dimensions, @package.cm
   end
 
-  def test_weight
+  test "#weight" do
     assert_equal @mass, @package.weight
     assert_instance_of Measured::Weight, @package.weight
   end
 
-  def test_weight_actual
+  test "#weight actual" do
```

Comment:
Something like `#weight for actual` would be more consistent with some of the above tests (e.g., `#centimetres for volume`). Similarly for the next couple of tests.